src/main/java/fr/techad/edc/httpd/SearchHandler.java

```diff
@@ -1,51 +1,94 @@
 package fr.techad.edc.httpd;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.BooleanUtils;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.fasterxml.jackson.databind.ObjectMapper;
+
 import fr.techad.edc.httpd.search.ContentSearcher;
 import fr.techad.edc.httpd.search.DocumentationSearchResult;
 import io.undertow.server.HttpHandler;
 import io.undertow.server.HttpServerExchange;
 import io.undertow.util.Headers;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.nio.ByteBuffer;
-import java.util.Collections;
-import java.util.Deque;
-import java.util.List;
-import java.util.Map;
 
 /**
- * TECH ADVANTAGE
- * All right reserved
- * Created by cochon on 24/04/2018.
+ * TECH ADVANTAGE All right reserved Created by cochon on 24/04/2018.
  */
 public class SearchHandler implements HttpHandler {
-    static final Logger LOGGER = LoggerFactory.getLogger(SearchHandler.class);
-    private final ObjectMapper objectMapper;
-    private final WebServerConfig config;
+  static final Logger LOGGER = LoggerFactory.getLogger(SearchHandler.class);
+  private final ObjectMapper objectMapper;
+  private final WebServerConfig config;
+
+  public SearchHandler(ObjectMapper objectMapper, WebServerConfig config) {
+    this.objectMapper = objectMapper;
+    this.config = config;
+  }
 
-    public SearchHandler(ObjectMapper objectMapper, WebServerConfig config) {
-        this.objectMapper = objectMapper;
-        this.config = config;
+  public void handleRequest(HttpServerExchange exchange) throws Exception {
+
+    Map<String, Deque<String>> queryParameters = exchange.getQueryParameters();
+    LOGGER.debug("Query Parameters: {}, Query: {}", queryParameters, exchange.getQueryString());
+
+    Deque<String> query = queryParameters.get("query");
+    Deque<String> language = queryParameters.get("lang");
+    Deque<String> limit = queryParameters.get("limit");
+    Deque<String> exact = queryParameters.get("exact");
+    Boolean exactMode = false;
+    String lang = "";
+    int limitResults = 100;
+    if (exact != null ) {
```

Comment:
See https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/BooleanUtils.html#toBoolean-java.lang.String-
if null, return false. So it is useless to check the value.